playbook.md

```diff
@@ -837,118 +820,81 @@ indicative of a wider problem. Or, if a single client sends lots of tickets
 within a short time, it might be wise to assign a delivery manager to oversee
 them all and ensure that the requests make sense when considered as a group.
 
-#### Assignments
+##### Assignments
 
 No matter what group you're in, when a ticket is assigned to you, you will
 receive an email. The ticket will also appear on your home screen in Zendesk.
 
 When you are assigned a ticket, it is your responsibility to review it and take
 action. You must either solve the ticket or pass it on to someone who can.
 
-A wide variety of issues can be reported through the helpdesk, so it's not
-possible to give an exhaustive guide on how to solve tickets. But there are some
-articles in the [Help Centre](https://dxw.zendesk.com) which may be useful.
-These are written for clients, but contain lots of useful information.
-
-It's important to be proactive about solving tickets. As soon as we receive a
-ticket, the clock starts ticking: clients expect us to be
-[responsive](#responsiveness-1) and deal with their problems promptly.
+A variety of issues can be reported through the helpdesk, so it's not possible to give an exhaustive guide on how to solve tickets. But there are some articles in the [Help Centre](https://dxw.zendesk.com) which may be useful. These articles are written for clients, but contain lots of useful information.
 
-If we're not sure how to proceed with a particular ticket, we don't just let it
-sit there: we ask for help. If we know what to do but don't have time to get it
-done, we get advice from a delivery manager.
+It's important to be proactive about solving tickets. As soon as we receive a ticket, the clock starts ticking. Clients expect us to be [responsive](#responsiveness-1) and deal with their problems promptly. If we're not sure how to proceed with a particular ticket, we don't just let it sit there, we ask for help from another member of the team.
```

Comment:
At some point in the past, this "responsive" anchor became broken. It should be `#be-responsive` instead of `#responsiveness-1`.

Reply:
Fixed in d4b1c06